app/views/shared/views/ems_common/_show.html.haml

```diff
@@ -12,5 +12,7 @@
       var miq_after_onload = "miqAsyncAjax('#{url_for(:action => @ajax_action, :id => @record)}');"
   - elsif @showtype == "dialog_provision"
     = render :partial => "shared/dialogs/dialog_provision"
+  - elsif @display == "main"
+    = render :partial => @showtype
```

Comment:
@h-kataria I saw that this is a common strategy in the `show.html.haml` files. Do you remember why it was missing here? Also it's even more common just:

    - else
      = render :partial => @showtype

Do you know why it wasn't possible here? I briefly tested it and it should work (I didn't check all the providers though).